src/main/groovy/org/transmartproject/core/dataquery/highdim/tworegion/JunctionValues.groovy

```diff
@@ -0,0 +1,16 @@
+package org.transmartproject.core.dataquery.highdim.tworegion
+
+/**
+ * Represents metadata about one junction of a two region variant
+ */
+interface JunctionValues {
+    long getUpEnd();
+    String getUpChromosome();
+    long getUpPos();
+    char getUpStrand();
+    long getDownEnd();
+    String getDownChromosome();
+    long getDownPos();
+    char getDownStrand();
+    boolean getIsInFrame();
```

Comment:
but I want to have a readonly property called isInFrame - will groovy use isInFrame() as an accessor for that?